docs/DafnyRef/Statements.md

```diff
@@ -928,7 +928,7 @@ has the same meaning as
 
 <!-- %no-check -->
 ```dafny
-if exists x :| P { var x :| P; S } else { T }
+if exists x :: P { var x :| P; S } else { T }
```

Comment:
Good catch!

Reply:
That's why as much code in docs as possible should be automatically checked...